idb/analysis.py

```diff
@@ -593,10 +593,13 @@ def pcb_version(self):
                 if self.len_sbytes == 119:
                     pass
                 # 7.20 <= ver <= 7.50?
-                elif self.len_sbytes in (141, 172):
+                elif self.len_sbytes in (141, 142, 172, 173):
                     jump_size = 1 if self.wordsize == 4 else 2
                     jump_size2 = 0 if self.wordsize == 4 else 1
 
+                    if self.len_sbytes in (142, 173):
+                        self.vsAddField("unknown-1", v_int8())
```

Comment:
recommend using an underscore rather than a slash, because vstruct will try to create an instance property with this name.

Reply:
I mean it just an index -1, because I don't want to resort the latter.